.pre-commit-config.yaml

```diff
@@ -3,4 +3,3 @@ repos:
   rev: stable
   hooks:
   - id: black
-    language_version: python3.6
```

Comment:
When using Python 3.7, with this line enabled the pre-commit fails. Since it's optional, I would remove and It shouldn't affect the behavior